_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -492,9 +492,7 @@ public static function get_plans( $request ) {
 	 * @return array An array of jitms
 	 */
 	public static function get_jitm_message( $request ) {
-		require_once( JETPACK__PLUGIN_DIR . 'class.jetpack-jitm.php' );
-
-		$jitm = Jetpack_JITM::init();
+		$jitm = new Automattic\Jetpack\JITM\Manager();
```

Comment:
We could do `use  Automattic\Jetpack\JITM\Manager as JITM_Manager` at the top of this file and then use
`$jitm = new JITM_Manager();` here.